src/app/personside/infotabs/dyplenkeTest/ytelserDyplenker.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import TestProvider from '../../../../test/Testprovider';
+import InfoTabs from '../InfoTabs';
+import { BrowserRouter } from 'react-router-dom';
+import { INFOTABS } from '../InfoTabEnum';
+import { getAktivTab, ytelserTest } from './utils';
+
+test('bytter til riktig tab og åpner valgt ytelse ved bruk av dyplenke fra oversikt', () => {
+    const infoTabs = mount(
+        <TestProvider>
+            <BrowserRouter>
+                <InfoTabs />
+            </BrowserRouter>
+        </TestProvider>
+    );
+
+    expect(getAktivTab(infoTabs)).toContain(INFOTABS.OVERSIKT);
+
+    clickOnYtelse(infoTabs);
+
+    expect(getAktivTab(infoTabs)).toContain(INFOTABS.YTELSER);
+
+    const apenYtelse = infoTabs.find('.ekspanderbartPanel--apen').html();
+    const expectedApenYtelse = infoTabs
+        .find('.' + ytelserTest.ytelse)
+        .last()
```

Comment:
Kanskje velge nummer to feks. Å velge første/siste virker litt mindre robust..?

Reply:
Det ser ut som det den ikke finner samme objekt på _at(2)_ ved knappetrykk og ved ytelseoversikt. Vi tar en titt på det når du er tilbake